dao/db.go

```diff
@@ -41,6 +44,19 @@ func Init() {
 	}
 	rawDB.SetMaxOpenConns(20)
 
+	// Perform database migrations
+	if conf.MigrationsReset {
+		DB.Exec(`DROP SCHEMA "public" CASCADE`)
+		DB.Exec(`CREATE SCHEMA "public"`)
+	}
+
+	if conf.MigrationsSetup || conf.MigrationsReset {
```

Comment:
I think I want the `setup` flag to just create the db - kinda like we do for `sources_api_test`. 

I want the migrations to just always run every time - since the app theoretically shouldn't be running if the db isnt migrated completely. This also prevents us from having to have an initContainer to run the migrations.